test/ctia/test_helpers/migration.clj

```diff
@@ -8,4 +8,6 @@
   [app]
   {:ConfigService (-> (helpers/get-service-map app :ConfigService)
                       (select-keys [:get-config
-                                    :get-in-config]))})
+                                    :get-in-config]))
+   :FeaturesService (-> (helpers/get-service-map app :FeaturesService)
+                        (select-keys [:flag-value]))})
```

Comment:
Since it's not used, I would rather open this schema.

Reply:
Looks like it's used here: https://github.com/threatgrid/ctia/blob/master/src/ctia/task/migration/store.clj#L707